### DISCUSSION TITLE
Help,I cannot launch Photok with the secret launch code after hiding the app

### DISCUSSION THREAD

**Discussion by leonlatsch** [leonlatsch/Photok#141] [General]
The hide app feature is flaky and just does not work on some devices.

## Good News
There is now a solution for this:

_With version **1.4.0** of Photok the Recovery Menu was introduced. If you want to access Photok again, and your code isn't working try this:_

* Enable Airplane Mode
* Press the fake Phone Icon of Photok
* Press the fake Phone Icon of Photok again within 5 seconds
* Now the recovery Menu should open. Here you can open the app without the dial code or reset the hide Photok setting completely.

**Comment by Recentlystarted** [comment]
That was a great push Leon. i am here as per your play store review comment. hope will get the solution ASAP. Keep Smiling ;)
